launchers/mdsp-connector/consumer.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM gradle:7.4.1-jdk17 AS build
+
+COPY --chown=gradle:gradle . /home/gradle/project/
+WORKDIR /home/gradle/project/
+RUN gradle :samples:other:file-transfer-http-to-http:consumer:shadowjar --no-daemon
+
+FROM openjdk:17-slim
+
+WORKDIR /app
+COPY --from=build /home/gradle/project/samples/other/file-transfer-http-to-http/consumer/build/libs/consumer.jar /app
+COPY --from=build /home/gradle/project/samples/other/file-transfer-http-to-http/consumer/config.properties /app
```

Comment:
It would be nice to be able to inject secrets inside without pushing them to github (e.g. oauth tokens) - possible solution in github secrets and just append to file

Reply:
They are not necessarily secrets in the config.properties. just additional settings not set in the java executable call. The secrets are handled differently via the Helm Chart.